From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: chwe17 <weberc18@gmail.com>
Date: Sun, 4 Aug 2019 18:05:50 -0400
Subject: [ARCHEOLOGY] Tinkerboard camera support (#1482)

> X-Git-Archeology: > recovered message: > * fix kernelcrash with config_of_overlay combined with isp1 camera driver,based on:
> X-Git-Archeology: > recovered message: > https://github.com/TinkerBoard/debian_kernel/commit/30a8401c2f3851f4e9b46c9d3e8e1138ce8d5b51
> X-Git-Archeology: > recovered message: > * add basic dtbo infrastucture
> X-Git-Archeology: > recovered message: > * set ov5647 (RPi cam rev 1.3) as default, imx219 (RPi cam V2.1) outputs a blurry green videostream at the moment
> X-Git-Archeology: > recovered message: > * maybe fix powering for camera
> X-Git-Archeology: > recovered message: > * adjust mainline dtbo patch to bsp kernel
> X-Git-Archeology: > recovered message: > * got you! little.. dtbo's are properly built and applied during boot.
> X-Git-Archeology: > recovered message: > * add imx219 overlay to switch between cameras & add ae/awb fix for ov 5647 from asus tinker repo
> X-Git-Archeology: > recovered message: > * patch imx219 up to https://github.com/TinkerBoard/debian_kernel/commit/96396323fa3d8941e4d556be83591333329d2799
> X-Git-Archeology: > recovered message: > * bring back wireguard, otherwise igor gets grumpy :D
> X-Git-Archeology: > recovered message: > * fix camera status
> X-Git-Archeology: - Revision a040785d4299e10255d87fdfcfa70b56e0b6779f: https://github.com/armbian/build/commit/a040785d4299e10255d87fdfcfa70b56e0b6779f
> X-Git-Archeology:   Date: Sun, 04 Aug 2019 18:05:50 -0400
> X-Git-Archeology:   From: chwe17 <weberc18@gmail.com>
> X-Git-Archeology:   Subject: Tinkerboard camera support (#1482)
> X-Git-Archeology:
> X-Git-Archeology: - Revision 150ac0c2afa147d9e3b036c8ecd8238fe5648cf3: https://github.com/armbian/build/commit/150ac0c2afa147d9e3b036c8ecd8238fe5648cf3
> X-Git-Archeology:   Date: Tue, 19 Nov 2019 23:25:39 +0100
> X-Git-Archeology:   From: Igor Pecovnik <igorpecovnik@users.noreply.github.com>
> X-Git-Archeology:   Subject: Remove K<4, change branches, new features (#1586)
> X-Git-Archeology:
> X-Git-Archeology: - Revision b2751ab56f2a0f8e4ff1e55b4de3a1fdf5647236: https://github.com/armbian/build/commit/b2751ab56f2a0f8e4ff1e55b4de3a1fdf5647236
> X-Git-Archeology:   Date: Tue, 29 Dec 2020 22:52:09 +0100
> X-Git-Archeology:   From: JMCC <JMCC@localhost>
> X-Git-Archeology:   Subject: rk3399's-legacy: Kernel patch to fix panic when RKISP1 is enabled
> X-Git-Archeology:
> X-Git-Archeology: - Revision 0cdffb29b07305209efb12cf3b5ac6032d3a1153: https://github.com/armbian/build/commit/0cdffb29b07305209efb12cf3b5ac6032d3a1153
> X-Git-Archeology:   Date: Wed, 24 Mar 2021 19:01:53 +0100
> X-Git-Archeology:   From: Igor Pecovnik <igorpecovnik@users.noreply.github.com>
> X-Git-Archeology:   Subject: Renaming DEV branch to EDGE (#2704)
> X-Git-Archeology:
---
 drivers/of/property.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/of/property.c b/drivers/of/property.c
index 2c201e2c257d..ae79a6f82b27 100644
--- a/drivers/of/property.c
+++ b/drivers/of/property.c
@@ -900,7 +900,7 @@ of_fwnode_graph_get_port_parent(struct fwnode_handle *fwnode)
 	struct device_node *np;
 
 	/* Get the parent of the port */
-	np = of_get_next_parent(to_of_node(fwnode));
+	np = of_get_parent(to_of_node(fwnode));
 	if (!np)
 		return NULL;
 
-- 
Armbian

